prodigy.el

```diff
@@ -581,10 +596,20 @@ the timeouts stop."
   "Return true if SERVICE is currently stopping, false otherwise."
   (eq (plist-get service :status) 'stopping))
 
+(defun prodigy--display-process-buffer (buffer)
```

Comment:
Is refactoring the logic for deciding which display function to use the way for point 3?